src/js/wizards/Sell/6_Margin/components/MarginSelectorForm.jsx

```diff
@@ -46,21 +53,19 @@ class MarginSelectorForm extends Component {
         <FormGroup className="mb-0">
           <Row>
             <Col md={9} sm={9} xs={8}>
-              <Slider className="mb-3 p-4" min={-99} max={100} defaultValue={0}
-                      onChange={(value) => this.onMarginChange(value)} value={margin}/>
+              <Slider className="mb-3 p-4" min={-100} max={100} defaultValue={1}
```

Comment:
-100% in this case would mean giving away the crypto for free :)

Reply:
Oops 🤦‍♂